setup.py

```diff
@@ -29,6 +29,7 @@ def get_install_requires() -> List[str]:
 
     requirements = [
         "alembic",
+        "click==7.1.2",
```

Comment:
How about adding a `TODO` note here to remove this dependency after the bug is fixed.

Reply:
Done.